src/main/java/core/basesyntax/Pair.java

```diff
@@ -1,5 +1,59 @@
 package core.basesyntax;
 
-class Pair {
+class Pair<T, S> {
+    T first;
```

Comment:
don't forget about access modifiers for your fields

Reply:
thank you. added